dev/firely/docker-compose.yaml

```
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  firely:
+    image: firely/server:${FHIR_IMAGE_TAG:-5.2.0}
+    environment:
+        VONK_Administration:Repository: SQLite
+        VONK_Administration:SQLiteDbOptions:ConnectionString: "Data Source=./data/vonkadmin.db"
+        VONK_Administration:SQLiteDbOptions:AutoUpdateDatabase: "true"
+    volumes:
+      - ./license/firelyserver-license.json:/app/firelyserver-license.json:ro
+      - ./settings/logsettings.instance.json:/app/logsettings.instance.json:ro
```

The Vonk admin SQLite DB should probably be on a docker volume, so it's not accidentally deleted (from the checkout)

```suggestion
      - ./settings/logsettings.instance.json:/app/logsettings.instance.json:ro
      - db-data:/app/data
volumes:
  db-data: {}
```